composer.json

```diff
@@ -88,9 +88,6 @@
         "thecodingmachine/phpstan-safe-rule": "^1.2.0"
     },
     "config": {
-        "platform": {
```

Comment:
I don't think removing the platform completely is the way, since we do want it. More likely we want to unset it when testing non-locked dependencies like done in https://github.com/infection/infection/pull/1786/files#diff-6e608e02c595d53ab6b70822a2bf19abcfc6ddcc976c2f536ad5bfca20f0443fR51-R53